dataclients/kubernetes/definitions/definitions.go

```diff
@@ -9,6 +9,16 @@ import (
 
 var errInvalidMetadata = errors.New("invalid metadata")
 
+type ListMetadata struct {
+	*Metadata
```

Comment:
I think ListMetadata does not embed standard metadata

Reply:
this will addressed in the next push